includes/Lister/GalleryList.php

```diff
@@ -50,12 +52,31 @@ class GalleryList extends Lister {
 	public function formatItem( Article $article, $pageText = null ) {
 		$item = $article->mTitle;
 
-		if ( $pageText !== null ) {
-			// Include parsed/processed wiki markup content after each item before the closing tag.
-			$item .= $pageText;
+		// If PageImages is loaded and we are not in the file namespace, attempt to assemble a gallery of PageImages
+		if ( $article->mNamespace != NS_FILE && ExtensionRegistry::getInstance()->isLoaded( 'PageImages' ) ) {
+			
+			$pageImage = $this->getPageImage( $article->mID ) ?: false;
+			
+			if ( $pageImage ) {
+				// Successfully got a pageimage, wrapping it
+				$item = $this->getItemStart() . $pageImage . "| [[" . $item . "]]" . $this->itemEnd . "link=" . $item . "";
+			}
+			else {
```

Comment:
Move this up one line for a bit nicer formatting so it looks like `} else {` on one line

Reply:
Done